frontend/src/App.tsx

```diff
@@ -83,7 +83,7 @@ import withScreencast, {
   ScreenCastHOC,
 } from "./hocs/withScreencast/withScreencast"
 
-// WARNING: order matters
```

Comment:
Think this is a residual of our css days. Don't believe order really matters anymore